third_party/BUILD

```diff
@@ -26,6 +26,7 @@ filegroup(
         "//third_party/re2:LICENSE",
         "//third_party/re2j:LICENSE",
         "//third_party/riegeli:LICENSE",
+        "//third_party/safe_html_types:LICENSE",
```

Comment:
Can you add a comment saying why we need this so we don't try to delete it again?

Reply:
PTAL